src/lib/audio/index.tsx

```diff
@@ -209,8 +209,9 @@ export class AudioPlayer {
         this.audio.currentTime = this.offset;
     }
 
-    rawSeek(position: number): void {
-        this.audio.currentTime = position;
+    rawSeek(position: number, useOffset = false): void {
+        const newTime = useOffset ? position + this.offset : position;
```

Comment:
I think worth adding a comment explaining when the offset would be used here - e.g. one of those module functions